aptos-move/framework/aptos-token-objects/sources/property_map.move

```diff
@@ -343,7 +343,41 @@ module aptos_token_objects::property_map {
         let constructor_ref = object::create_named_object(creator, b"");
         let object = object::object_from_constructor_ref<object::ObjectCore>(&constructor_ref);
 
-        let input = end_to_end_input();
+        let input = prepare_input(
```

Comment:
Is this a workaround for our V2 test_only handling?  If so, add a TODO comment to change it back when we fix it, or leave it out of this PR.

Reply:
No, please see the PR description for more on this.

A previous PR by Wolfgang created a workaround and created this issue: https://github.com/aptos-labs/aptos-core/issues/11724, I am reverting code to before that hack because we no longer need it due to the optimization.

Reply:
done